styleguide/components/00-settings/_colors.scss

```diff
@@ -75,7 +75,8 @@ $tints: (
     @warn "`#{inspect($tint)}` doesn't seem to be a valid tint value.";
   }
 
-  @return mix(map-get($colors, $base-color), $color-white, map-get($tints, $tint));
+  // @return mix(map-get($colors, $base-color), $color-white, map-get($tints, $tint));
```

Comment:
Let's remove this inline code comment ;-) Yeay git history!

Reply:
Done.